tests/test_ABCD.py

```diff
@@ -97,16 +104,21 @@ def test_fixed_poi_fit():
 def test_bkg_only_fit():
     abcd = ABCD(observed_yields, signal_yields, signal_uncertainty)
     bkg_only_fit = abcd.bkg_only_fit()
-    assert bkg_only_fit[0][0] == 0
-    assert bkg_only_fit[1][0] == 0
-    assert math.isclose(bkg_only_fit[2][0], observed_yields['A'], rel_tol=1e-2)
+    par_names = abcd.model.config.par_names
+    assert bkg_only_fit[par_names.index('mu')][0] == 0
```

Comment:
The motivation for using parameter names over parameter list indexes is because of a change in parameter sorting in `pyhf` `v0.7.0`:

```
# v0.6.3

(Pdb) bkg_only_fit
array([[ 0.        ,  0.1       ],
       [ 0.        ,  0.1       ],
       [10.        ,  2.22445581],
       [ 2.        ,  0.38670796],
       [ 3.        ,  0.63131606]])

(Pdb) abcd.model.config.par_names()
['mu', 'systematic_uncertainty', 'mu_b', 'tau_B', 'tau_C']
(Pdb) abcd.model.config.par_order
['mu', 'systematic_uncertainty', 'mu_b', 'tau_B', 'tau_C']

# v0.7.0

(Pdb) bkg_only_fit
array([[10.        ,  2.22429436],
       [ 0.        ,  0.        ],
       [ 2.        ,  0.38669598],
       [ 3.        ,  0.63127644],
       [ 0.        ,  0.        ]])

(Pdb) abcd.model.config.par_names
['mu_b', 'mu', 'tau_B', 'tau_C', 'systematic_uncertainty']
(Pdb) abcd.model.config.par_order
['mu_b', 'mu', 'tau_B', 'tau_C', 'systematic_uncertainty']
```